CONTRIBUTING.md

```diff
@@ -153,3 +153,49 @@ Before merging a PR make sure:
 
 Merge the PR by using "Squash and merge" option on Github. Avoid creating merge commits.
 After the merge make sure referenced issues were closed.
+
+### Developing guidelines
+
+#### Building [OCI Images](https://github.com/opencontainers/image-spec/blob/master/spec.md) for multiple arch (linux/arm64, linux/amd64)
+
+OCI images could be built and published by [buildx](https://github.com/docker/buildx), it could be executed for local test via:
+
+```
+$ OPERATOR_VERSION=devel ./.ci/publish-images.sh
+```
+
+more arch support only need to change `--platform=linux/amd64,linux/arm64`
+
+if we want to execute this in local env, need to setup buildx:
+
+0. ensure docker ready
+
+```
+$ curl https://raw.githubusercontent.com/docker/docker/master/contrib/check-config.sh > check-config.sh
```

Comment:
Can we just go without this? If this is part of the official instructions, then let's just point users there.

Reply:
sure.